src/java.base/share/classes/java/lang/foreign/Linker.java

```diff
@@ -704,6 +731,8 @@ static Option firstVariadicArg(int index) {
          * }
          * }
          *
+         * @apiNote This linker option can not be combined with {@link #critical}.
+         *
```

Comment:
That seems more specification (that can be asserted on) then an informative note.

Reply:
True. I fold it into the main spec body.